kythe/release/kubernetes/buildkite/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+#
+# Copyright 2019 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+CLUSTER=buildkite-agent-cluster
```

Comment:
Document this script.  When is it called?

Reply:
This is the 0th task required for talking to a Kubenetes cluster as documented by k8s, but added a comment.